erdgo/utils.go

```diff
@@ -31,3 +39,29 @@ func SignTransaction(tx *Transaction, privateKey []byte) error {
 
 	return nil
 }
+
+// SignTransactionHash signs a transaction with the provided private key
```

Comment:
add more details about this method in comment

Reply:
Done